tools/minify-js

```diff
@@ -45,11 +45,11 @@ def get_changed_source_files(other_checkout):
     try:
         git_dir = os.path.join(other_checkout, '.git')
         old_commit_sha1 = subprocess.check_output(['git', 'rev-parse', 'HEAD'],
-                                                  env={'GIT_DIR': git_dir})
+                                                  env={'GIT_DIR': git_dir}, universal_newlines=True)
```

Comment:
Hmm, we should probably match what hexdigest does, so if it does text, let's stick with this.  Thanks for investigating!